### PR TITLE
Added "Sponsor section" partial into attend and incompany page

### PR DESCRIPTION
**Pull request by FlorisSp**
screenshot from 2017-03-30 10-55-36

### CODE REVIEW

**Comment by JTBreunissen** [comment]
i resolved the conflict in the attend.html

File: app/views/pages/attend.html.erb [hunk 1]
```diff
@@ -1,6 +1,5 @@
-<h1>Pages#attend</h1>
-<p>Find me in app/views/pages/attend.html.erb</p>
-
 <%= render "shared/what_will_you_learn" %>
 <%= render "shared/videotestimonial" %>
 <%= render "shared/testimonials" %>
+<%= render "shared/sponsor_section" %>
+
```
Comment (arnoFleming) [inline app/views/pages/attend.html.erb]: remove trailing newlines!

File: app/views/shared/_sponsor_section.html.erb [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<section>
+  <div class="container marketing">
+    <div class="row">
```
Comment (arnoFleming) [inline app/views/shared/_sponsor_section.html.erb]:
```
container
  row
    col-md-4
    col-md-4
    col-md-4

    col-md-4  
    col-md-4
    col-md-4
      container
        row
          col-xs-4 image
          col-xs-8 text
```

Comment (arnoFleming) [inline app/views/shared/_sponsor_section.html.erb]: please update

File: db/schema.rb [hunk 3]
```diff
@@ -15,23 +15,21 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "events", force: :cascade do |t|
```
Comment (arnoFleming) [inline db/schema.rb]: This should not be here - its a PR about partials. Furthermore, you should not change your schema unless you have migrations in your PR

**Comment by arnoFleming** [comment]
please rebase

File: app/views/pages/attend.html.erb [hunk 4]
```diff
@@ -1,6 +1,4 @@
-<h1>Pages#attend</h1>
-<p>Find me in app/views/pages/attend.html.erb</p>
-
 <%= render "shared/what_will_you_learn" %>
 <%= render "shared/videotestimonial" %>
 <%= render "shared/testimonials" %>
+<%= render "shared/sponsor_section" %>
```
Comment (arnoFleming) [inline app/views/pages/attend.html.erb]: please rename to '/shared/sponsor' - its irrelevant if it is a section

File: app/views/shared/_sponsor_section.html.erb [hunk 5]
```diff
@@ -0,0 +1,76 @@
+<div class="container ">
```
Comment (arnoFleming) [inline app/views/shared/_sponsor_section.html.erb]: and, btw it isn't a section... Add that :D

File: db/schema.rb (hunk 3 above)
Comment (arnoFleming) [inline db/schema.rb]: why is this still here?

File: spec/views/shared/_sponsor_section.html.erb_spec.rb [hunk 6]
```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+describe "shared/_sponsor_section.html.erb" do
+  it "contains content" do
+    render
+    rendered.should have_text("Proudly supported by")
```
Comment (arnoFleming) [inline spec/views/shared/_sponsor_section.html.erb_spec.rb]: don't use `should`, it is from rspec 2.x, use the rspec 3.x version format:  
```
expect(thing).to have_some('whatever')
```

**Pull request merged**